src/commands/metadata-templates/index.js

```diff
@@ -4,7 +4,7 @@ const BoxCommand = require('../../box-command');
 
 class MetadataTemplatesListCommand extends BoxCommand {
 	async run() {
-		const { flags, args } = this.parse(MetadataTemplatesListCommand);
+		await this.parse(MetadataTemplatesListCommand);
```

Comment:
do we need to parse it at all?

Reply:
After brief discussion: We're not sure if it would affect anything (if this function is only parsing input params). We decided to keep it for now.